src/Components/Projects/Projects.tsx

```diff
@@ -74,15 +74,15 @@ const handleClose = () => setModalState("close");
         </div>
        </Modal.Body>
      </Modal>
- <div className="container bg-white rounded-3 px-3">
+ <div className="container bg-white rounded-3 p-3">
   <div className="col-md-12 trans-head">
     <div className="top d-flex justify-content-between ">
       <h4>Projects</h4>
       <button onClick={navToAddPro} className='btn btn-warning bg-warning text-white'> <i className='fa-plus'></i> Add New Project </button>
     </div>
   </div>
   <div className="col-md-12 ">
-    <input type="text" placeholder='SearchFleets 'className='rounded-4 border-1 mb-4 pro' />
+    <input type="text" placeholder='    SearchFleets 'className='rounded-4 border-1 mb-4 pro' />
```

Comment:
The search is not implemented yet, but the place holder text should reflect what query will be used in the searching such as:
Search By Task Title